src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,21 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    private Bulldozer bulldozer1 = new Bulldozer();
+    private Bulldozer bulldozer2 = new Bulldozer();
+    private Bulldozer bulldozer3 = new Bulldozer();
+    private List<Bulldozer> bulldozers = List.of(bulldozer1, bulldozer2, bulldozer3);
+
+    @Override
+    public List<Bulldozer> get() {
+        List<Bulldozer> listOfBulldozers = new ArrayList<Bulldozer>();
+        for (Object element : bulldozers) {
+            listOfBulldozers.add((Bulldozer) element);
+        }
+        return listOfBulldozers;
+    }
```

Comment:
```suggestion
    @Override
    public List<Bulldozer> get() {
        return List.of(new Bulldozer(), new Bulldozer(), new Bulldozer());
    }
```
if you opt to use List.of, we can do it way simpler